packages/stacked/example/pubspec.yaml

```diff
@@ -49,7 +49,7 @@ dev_dependencies:
   # dependency injection
   build_runner:
   stacked_generator:
-    # path: ../../stacked_generator
+    path: ../../stacked_generator
```

Comment:
Revert this back or use dependency_override to use path in development and the actual package in production